R-package/src/cmake/modules/FindLibR.cmake

```diff
@@ -41,20 +41,19 @@ function(create_rlib_for_msvc)
     message(FATAL_ERROR "LIBR_CORE_LIBRARY, '${LIBR_CORE_LIBRARY}', not found")
   endif()
 
-  find_program(GENDEF_EXE gendef)
   find_program(DLLTOOL_EXE dlltool)
 
-  if(NOT GENDEF_EXE OR NOT DLLTOOL_EXE)
-    message(FATAL_ERROR "Either gendef.exe or dlltool.exe not found!\
+  if(NOT DLLTOOL_EXE)
+    message(FATAL_ERROR "dlltool.exe not found!\
       \nDo you have Rtools installed with its MinGW's bin/ in PATH?")
   endif()
 
   set(LIBR_MSVC_CORE_LIBRARY "${CMAKE_CURRENT_BINARY_DIR}/R.lib" CACHE PATH "R.lib filepath")
 
-  # extract symbols from R.dll into R.def and R.lib import library
-  execute_process(COMMAND ${GENDEF_EXE}
-    "-" "${LIBR_CORE_LIBRARY}"
-    OUTPUT_FILE "${CMAKE_CURRENT_BINARY_DIR}/R.def"
+  execute_process(
+    COMMAND "${LIBR_HOME}/bin/Rscript"
```

Comment:
I think we should search for `Rscript` similarly to `LIBR_EXECUTABLE`.

Reply:
ok sure! good idea

Reply:
Take a look at the changes I just pushed to https://github.com/microsoft/LightGBM/commit/fafac03de0b44f19a600a9fb60776dd3765731ac

Instead of repeating the search everywhere, I'm relying on the fact that `R` and `Rscript` are always installed in the same location.

![image](https://user-images.githubusercontent.com/7608904/82830315-fa413e80-9e7a-11ea-97ab-7fb049661296.png)